ddp-workspace/projects/ddp-sdk/src/lib/services/picklistSortingPolicy.service.ts

```diff
@@ -0,0 +1,49 @@
+import { ActivityPicklistNormalizedGroup, ActivityPicklistOption } from 'ddp-sdk';
+import { SortOrder } from './sortOrder';
+import { Injectable } from '@angular/core';
+
+@Injectable({
+    providedIn: 'root',
+})
+export class PicklistSortingPolicy {
+    constructor(readonly mainSortOrder: SortOrder = SortOrder.NONE, readonly lastStableId?: string) {}
+
+    public sortPicklistGroups(groups: ActivityPicklistNormalizedGroup[]): ActivityPicklistNormalizedGroup[] {
+        const groupsCopy = groups.slice();
+        if (this.mainSortOrder === SortOrder.ALPHABETICAL) {
+            const sortedPicklistGroups = groupsCopy.sort(this.buildComparator('name'));
+            sortedPicklistGroups.forEach(group => group.options = this.sortPicklistOptions(group.options));
+        }
+        return groupsCopy;
```

Comment:
SortOrder.NONE should just result in copy of options in exactly same order. Otherwise we sort stuff.